python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -106,6 +108,21 @@ def parameters(self, include_sublayer=True):
                 params = self._args[0]._parameters
         return params
 
+    def buffers(self, include_sublayer=True):
+        """
+        Returns VarBase buffers of decorated layers. If set `include_sublayer` True,
+        the VarBase buffers created in sub layers will be added.
+        """
+        buffers = collections.OrderedDict()
+        if self.is_method():
+            if include_sublayer:
+                buffers = self._args[0].buffers()
+                names = [buffer.name for buffer in buffers]
+                buffers = collections.OrderedDict(zip(names, buffers))
+            else:
+                buffers = self._args[0]._buffers
+        return buffers
```

Comment:
Better more readable code as we discussed offline.

Reply:
Thanks, done.